programs/drift/src/state/user.rs

```diff
@@ -1444,9 +1641,77 @@ impl Size for UserStats {
 }
 
 impl UserStats {
-    pub fn update_maker_volume_30d(&mut self, quote_asset_amount: u64, now: i64) -> DriftResult {
+    pub fn get_fuel_bonus_numerator(
+        self,
+        last_fuel_bonus_update_ts: i64,
+        now: i64,
+    ) -> DriftResult<i64> {
+        let since_last = now.safe_sub(last_fuel_bonus_update_ts)?;
+        Ok(since_last)
+    }
+
+    pub fn update_fuel_bonus_trade(&mut self, fuel_taker: u32, fuel_maker: u32) -> DriftResult {
+        self.fuel_taker = self.fuel_taker.saturating_add(fuel_taker);
+        self.fuel_maker = self.fuel_maker.saturating_add(fuel_maker);
+
+        Ok(())
+    }
+
+    pub fn update_fuel_bonus(
+        &mut self,
+        fuel_deposits: u32,
+        fuel_borrows: u32,
+        fuel_positions: u32,
+        _now: i64,
+    ) -> DriftResult {
+        self.fuel_deposits = self.fuel_deposits.saturating_add(fuel_deposits);
+        self.fuel_borrows = self.fuel_borrows.saturating_add(fuel_borrows);
+        self.fuel_positions = self.fuel_positions.saturating_add(fuel_positions);
+        // self.last_fuel_bonus_update_ts = now;
```

Comment:
why'd you need to comment this out? kinda weird there is an increment and update fuel bonus fn